### DISCUSSION TITLE
Cross platform Compiler Toolkit

### DISCUSSION THREAD

**Discussion by MuradMathematics** [odin-lang/Odin#974] [Ideas/Requests]
(Feature request) Having a cross platform compiler would be great. As I see, so far every platform depends on their own linker (Windows - MSVC, Mac: XCode, etc). Having all integrated into one software solution would be a better approach. However this might not have a high priority for the language itself (atleast at this stage of development).

**Comment by doongjohn** [comment]
is there any update on this topic?

**Comment by xzores** [comment]
would like to see a c to odin cross compiler.

**Comment by Platin21** [comment]
Well you can technically already link with lld which to my knowlegde can do full cross platform binaries. The issue will be more that the sysroot and everything has to be there in order to create a usable binary for certain things.
